src/cacheout/cache.py

```diff
@@ -74,6 +91,7 @@ class Cache:
         default: Default value or function to use in :meth:`get` when key is not found. If callable,
             it will be passed a single argument, ``key``, and its return value will be set for that
             cache key.
+        on_get: Callback which will be executed when a cache entry is retrieved.
         on_delete: Callback which will be executed when a cache entry is removed.
```

Comment:
My thinking of putting the variable descriptions here was so that it would be prominent when viewing the class documentation. Does any of the details from the constant docstrings show up here in the generated docs?

Reply:
I guess we could add a class reference so that there's at least a link to the variable docs using something like:

```
See :class:`T_ON_DELETE_CALLBACK`.
```

Reply:
> My thinking of putting the variable descriptions here was so that it would be prominent when viewing the class documentation. Does any of the details from the constant docstrings show up here in the generated docs?

Yes, it works.